### PR TITLE
feat(client): Add markdown component

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by nimish-gupta — CHANGES_REQUESTED** [review]
Gif?

**Comment by piyushagwl007** [comment]
Common markdown options are supported.    
Also, you can embed youtube, Vimeo, loom using shared link.

Other sites can be embedded using the direct embed link.
Markdown

**Comment by piyushagwl007** [comment]
> Gif?

@nimish-gupta added the gif :smile: 

Sorry, I forgot it before asking for a review. Won't happen again.

File: client/src/components/markdownRenderer/index.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        source={data}
+        renderers={renderers}
+        plugins={plugins}
+      />
+    );
+  }
+}
+
+
+export default Markdown;
```
Comment (nimish-gupta) [inline client/src/components/markdownRenderer/index.jsx:48]: test for checking error handling

**Review by nimish-gupta — APPROVED** [review]
LGTM after the tests

**Review by karansapolia — APPROVED** [review]
LGTM. Ensure you fill the PR checklist completely everytime :slightly_smiling_face:

**Comment by piyushagwl007** [comment]
@nihalanisumit can you review this P.R. ?

**Review by nihalanisumit — APPROVED** [review]
LGTM

**Pull request merged**